cpp/notebooks/ros1_service_client.ipynb

```diff
@@ -1,21 +1,37 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "64c1bbf9-59e8-4394-99ed-f6b67661a120",
+   "metadata": {},
+   "source": [
+    "# **Tutorial to explore ROS 1 Service Client**"
```

Comment:
Remove Client, so just "ROS1 Service". Since, it contains both server and client.

Also the ipynb name, and link to the ipynb in README